app/utilities/schema.py

```diff
@@ -30,11 +30,12 @@
 
 
 @lru_cache(maxsize=None)
-def get_schema_path_map(dirs: Optional[Tuple[str, ...]] = None) -> Mapping:
+def get_schema_path_map(include_test_schemas: Optional[bool] = False) -> Mapping:
+    dirs = DEFAULT_SCHEMA_DIRS if include_test_schemas else SCHEMA_DIR
```

Comment:
What do you think about having a`TEST_SCHEMA_DIR` variable rather than `DEFAULT_SCHEMA_DIRS` and creating the dirs list here?

Reply:
Dir is now created in-place as it is not used elsewhere, however I still used module level variables for the dirs.